src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -606,6 +606,11 @@
 			<method signature="System.Void .ctor()" />
 		</type>
 
+		<!-- threads.c (abandon_mutexes) -->
```

Comment:
It is true that it handles just abandoned mutexes today, but I think you will likely add more to it (e.g. if you adopt managed Thread.Join). I would call this `exited_threads` and not imply that it is just for abadoned mutexes.